03-terraform-outputs-across-stacks/.gitignore

```diff
@@ -0,0 +1,37 @@
+# Local .terraform directories
```

Comment:
Can we move the content of this file to https://github.com/terramate-io/terramate-examples/blob/main/.gitignore instead of having a second .gitignore in this repo?

Reply:
Done